core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -949,7 +949,7 @@ protected function generateButtons($arrRow, $strTable, $arrRootIds=array(), $bln
 					}
 					else
 					{
-						$href = $this->addToUrl($v['href'] . '&amp;id=' . $arrRow['id'] . (Input::get('nb') ? '&amp;nc=1' : ''));
+						$href = $this->addToUrl(($v['href'] ?? '') . '&amp;id=' . $arrRow['id'] . (Input::get('nb') ? '&amp;nc=1' : ''));
```

Comment:
Should we maybe change line 950 from `else` to `elseif (!empty($v['href']))` instead?

Reply:
But `addToUrl` also adds the `REQUEST_TOKEN` - even with empty input.
This could be a BC break?

Reply:
You are right. 👍